extensions/KHR/EGL_KHR_platform_wayland.txt

```diff
@@ -93,6 +94,22 @@ New Behavior
     until the next buffer swap. The rationale behind this behavior is to keep
     operations result accurate until the next swap.
 
+    Add the following text to subsection 3.10.1 titled "Posting to a
+    Window" after paragraph 3 of the description of eglSwapBuffers:
+
+    "When the <surface> refers to a Wayland platform surface, synchronously
+    within the call to eglSwapBuffers the EGL implementation will invoke a
+    series of requests on the associated wl_surface, including attaching the
+    new front buffer, a full surface damage via wl_surface.damage or
+    wl_surface.damage_buffer, and finally a wl_surface.commit."
+
+    If the EGL implementation associated a wp_linux_drm_syncobj_surface_v1
+    object with the surface, an acquire and a release point will be associated
+    with the surface prior to the commit as well.
+
+    An EGL implementation must not create a wp_color_management_surface_v1
```

Comment:
I don't think that's realistic. Don't we need this to implement things like EGL_EXT_gl_colorspace_scrgb_linear properly?

Reply:
That sounds like a necessity indeed, unless the application itself handles describing the content, and EGL just generating appropriate pixel values. Would EGL need to manage this state internally, I guess it needs to be specified when exactly it will do so, to not cause conflicts with existing color manager protocol users that uses it to describe EGL content.

Reply:
In Vulkan we worked out a design where the PASSTHROUGH colorspace indicated the Vulkan implementation wouldn't create the Wayland object. Perhaps something similar is possible in EGL? I would expect the application isn't using a particular colorspace value currently if it's using the color management protocol directly. I haven't reviewed the EGL HDR/colorspace-related extensions in detail. I think the sRGB/linear colorspace spec has implications on how core rendering to the surface works, but I don't think the others do.